userspace/ksud/src/debug.rs

```diff
@@ -39,8 +39,9 @@ fn set_kernel_param(size: u32, hash: u32) -> Result<()> {
 }
 
 fn get_apk_path(package_name: &str) -> Result<String> {
-    let cmd = format!("pm path {}", package_name);
-    let output = Command::new("sh").arg("-c").arg(cmd).output()?;
+    let output = Command::new("cmd")
```

Comment:
better to use `pm path` becuase `cmd package path` is not available below Android 9 (http://aospxref.com/android-9.0.0_r61/xref/frameworks/base/services/core/java/com/android/server/pm/PackageManagerShellCommand.java#143)